delft/textClassification/wrapper.py

```diff
@@ -84,6 +85,13 @@ def __init__(self,
             if transformer_name is not None:
                 model_name += "_" + transformer_name
 
+        if learning_rate is None:
+            if transformer_name is None:
+                learning_rate = 0.0001
```

Comment:
Same remark as for sequence labeling, the default learning is divided by 10 for RNN ?

Reply:
Same as before, I got confused with alle these zeroes